docs/general/decode_a_cairo_trace.md

```diff
@@ -89,7 +89,7 @@ from starkware.cairo.lang.vm.reconstruct_traceback import reconstruct_traceback
 
 build_dir = Path("build")
 programs = {}
-for contract_name in ["externally_owned_account", "kakarot", "contract_account", "EVM"]:
+for contract_name in ["uninitialized_account", "kakarot", "generic_account", "EVM"]:
     artifact = build_dir / f"{contract_name}.json"
```

Comment:
remove `"contract_account"` as well right?

Reply:
indeed, spotted that already